blueprints/component/index.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env node */
+// Note - this is the exact same as ember's normal component index.js
```

Comment:
I haven't dug deep enough into this PR yet so this may be a premature question, but if this content is the same as what is already present why do we need this file?